File: scala/src/test/scala/freesim/FreesimTest.scala
package freesim

import org.junit.runner.RunWith
import cucumber.api.junit.Cucumber
import cucumber.api.junit.Cucumber.Options

@RunWith(classOf[Cucumber])
@Options(features = Array("classpath:freesim"), glue = Array("freesim.steps"))
class FreesimTest

File: java/src/test/java/freesim/FreesimTest.java
package freesim;

import cucumber.api.junit.Cucumber;
import org.junit.runner.RunWith;

@RunWith(Cucumber.class)
@Cucumber.Options(features = {"classpath:freesim"}, glue = {"freesim.steps", "core.hooks"}, format = {"pretty", "html:target/cucumber-html-report"})
public class FreesimTest {
}
